### ISSUE TITLE
Can't build on mac

### ISSUE THREAD

**Issue by avcarr2** [smith-chem-wisc/mzLib#629]
Since the recent change in the structure of the code base, I can no longer build on MacOS. I am getting a NETSDK1073: The FrameworkReference 'Microsoft.WindowsDesktop.App.WPF' was not recognized (NETSDK1073) (mzPlot). 

Obviously, the issue is that the WPF that mzPlot uses isn't compatible with mac, but in the past I haven't had any issue building or running tests from mzLib. 

Anybody know of any workarounds on this? I'll also post something if I can figure it out.

**Comment by acesnik** [comment]
I get the same error. I do see it before and after restructuring the base directory, though.

<img width="989" alt="image" src="https://user-images.githubusercontent.com/16342951/168859568-0e0d4f48-6b5a-467f-973e-dd9e127884fc.png">

**Comment by acesnik** [comment]
Possible workarounds:
* The `.dlls` are constructed for the other projects, so a workaround might be just copying them to your build directory of whatever other project you're working on. 
* You could also delete the mzPlot project from the `.sln` file on your clone

**Comment by avcarr2** [comment]
I tried deleting the mzPlot project, but that didn't completely work. It resolved the error originating in mzPlot, but I ended up getting the same error except localized to the Test project instead. Still working through that one.

**Comment by acesnik** [comment]
Could probably make a different configuration for Mac (right now, there's Release and Debug), that disincludes the WPF-dependent projects. Then you could specify `dotnet build --configuration NoWPF mzLib/mzLib.sln`

**Comment by avcarr2** [comment]
I made a different configuration for Mac that didn't reference mzPlot, and it appears that did the trick. I'll make a pull request to for the new special configuration.

**Comment by avcarr2** [comment]
After some experimenting with different configuration I'm still getting the issue. I'll keep researching the issue, but for now, the only thing that I've found to work is purging all references to mzPlot.

**Comment by acesnik** [comment]
Making mzPlot a separate library makes sense to me, too.

I think there will be some changes in the nuspec file, too, e.g. removing OxyPlot.

**Comment by trishorts** [comment]
is oxyplot connected to mzplot?

**Comment by acesnik** [comment]
Yes, I believe it's a dependency of only mzPlot, https://github.com/smith-chem-wisc/mzLib/search?q=oxyplot

**Comment by trishorts** [comment]
maybe we should have a separate nuget "mzGUI" for things that might just be for windows gui that could screw some other type stuff up?
